pkg/proxyserver/controllers/manifests.go

```diff
@@ -26,6 +26,10 @@ func newOwnerReference(config *proxyv1alpha1.ManagedProxyConfiguration) metav1.O
 
 func newServiceAccount(config *proxyv1alpha1.ManagedProxyConfiguration) *corev1.ServiceAccount {
 	return &corev1.ServiceAccount{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
they're applied to the hub cluster using direct create/update api calls, only those applied by the manifest-work will need the explicit type meta, is it?

Reply:
yeah, removed